test/Worker.Extensions.Tests/ServiceBus/ServiceBusSessionMessageConverterTests.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Can you also add E2E tests? There is a service bus [emulator](https://learn.microsoft.com/en-us/azure/service-bus-messaging/overview-emulator) now (I think we were waiting for this to exist).

You can create a service bus function here: https://github.com/Azure/azure-functions-dotnet-worker/tree/main/test/E2ETests/E2EApps/E2EApp

Then write code to test it here: https://github.com/Azure/azure-functions-dotnet-worker/tree/main/test/E2ETests/E2ETests

And finally, this script will need to update to setup the emulator: https://github.com/Azure/azure-functions-dotnet-worker/blob/main/setup-e2e-tests.ps1

This is a big ask so I am okay with this being setup in a new PR

Reply:
> This is a big ask so I am okay with this being setup in a new PR

I think we should have a separate issue for adding E2E tests here, it is definitely a large work item.

Reply:
I can open an issue for this to follow up on!

Reply:
https://github.com/Azure/azure-functions-dotnet-worker/issues/2904